.github/scripts/generate_ci_workflows.py

```diff
@@ -67,11 +50,11 @@ def gen_root_job_condition(self) -> None:
         # Once fully rollout, we can have strict constraints
         # e.g. ADD      env.GITHUB_ACTOR == '{self.trigger_actor}
         #      REMOVE   github.event.action !='{self.trigger_action}'
-        label_conditions = [f"github.event.action == '{self.trigger_action}'"] + \
-            [f"contains(github.event.pull_request.labels.*.name, '{label}')" for label in self.labels]
+        label_conditions = [
+            f"contains(github.event.pull_request.labels.*.name, '{label}')" for label in sorted(self.labels)]
         self.root_job_condition = f"(github.event_name != 'pull_request') || " \
             f"(github.event.action !='{self.trigger_action}') || " \
-            f"({' && '.join(label_conditions)})"
+            f"({' || '.join(label_conditions)})"
```

Comment:
The label design is a logical operator OR, i.e. adding ('ciflow/cuda' + 'ciflow/win') will trigger the union of them. (design feedback is needed here)